msbuild/Xamarin.MacDev.Tasks.Core/Tasks/ParseBundlerArgumentsTaskBase.cs

```diff
@@ -55,6 +61,7 @@ public override bool Execute ()
 			if (!string.IsNullOrEmpty (ExtraArgs)) {
 				var args = CommandLineArgumentBuilder.Parse (ExtraArgs);
 				List<string> xml = null;
+				List<string> custom_link_flags = null;
```

Comment:
Why do we use snake_case here and camelCase in the other vars?

Reply:
Fixed!